haas/api.py

```diff
@@ -423,6 +423,39 @@ def node_detach_network(node, nic, network):
     return '', 202
 
 
+@rest_call('PUT', '/TPM_metadata/<tpm_metadata>', Schema({
```

Comment:
Agree. Suggestion: use URIs for the standardized keys, much like we do with switch types; this solves the namespacing mechanism.